cob_hardware_config/robots/cob4-7/urdf/cob4-7.urdf.xacro

```diff
@@ -66,6 +72,14 @@
     <origin xyz="${head_cam_x} ${head_cam_y} ${head_cam_z}" rpy="${head_cam_roll} ${head_cam_pitch} ${head_cam_yaw}" />
   </xacro:usb_cam>
```

Comment:
as long as head is disabled in robot.xml, you might need to set the dof's to false...otherwise it will collapse in simulation
...same for sensorring!

Reply:
again components broken, to be clarified.. but if the sim will not  work I can temporally update the URDF

Reply:
:+1:
for explanation:
in simulation the head will hang loose because there is no controller because they are not loaded because the head part is commented in the robot.xml
setting the dof's to false makes the joints fixed instead of revolute and thus no controllers are needed